### PR TITLE
CFY-4110 optimized input files and added RHEL plugins

### PR DESCRIPTION
**Pull request by nir0s**

### CODE REVIEW

File: aws-ec2-RHEL-manager-blueprint-inputs.yaml [hunk 1]
```diff
@@ -123,36 +123,33 @@ management_worker_rpm_source_url: 'https://gigaspaces-repository-eu.s3.amazonaws
...
-# blueprint might need, could be upload trough this mechanism.
+# will reside on the manager and enable offline deployment. `dsl_resources`
+# should contain any resource needed in the parsing process (i.e. plugin.yaml files)
+# and any plugin archive should be compiled using the designated wagon tool
+# which can be found at: http://github.com/cloudify-cosmo/wagon.
+# The path should be passed to plugin_resources. Any resource your
+# blueprint might need, could be uploaded using this mechanism.
+
 #dsl_resources:
 #  - {'source_path': '/cfy/cloudify/plugins/fabric-plugin/plugin.yaml', 'destination_path': '/fabric-plugin/1.3rc1/plugin.yaml'}
```
Comment (ran-z) [inline aws-ec2-RHEL-manager-blueprint-inputs.yaml]: source links should point to S3, not file path. maxim changed these in the manager blueprints themselves (see here https://github.com/cloudify-cosmo/cloudify-manager-blueprints/blob/master/aws-ec2-manager-blueprint.yaml#L382 ) but apparently forgot in the inputs files.

File: openstack-manager-blueprint-inputs.yaml [hunk 2]
```diff
@@ -188,50 +188,33 @@
...
+# which can be found at: http://github.com/cloudify-cosmo/wagon.
+# The path should be passed to plugin_resources. Any resource your
+# blueprint might need, could be uploaded using this mechanism.
+
 #dsl_resources:
 #  - {'source_path': '/cfy/cloudify/plugins/fabric-plugin/plugin.yaml', 'destination_path': '/fabric-plugin/1.3rc1/plugin.yaml'}
 #  - {'source_path': '/cfy/cloudify/plugins/script-plugin/plugin.yaml', 'destination_path': '/script-plugin/1.3rc1/plugin.yaml'}
 #  - {'source_path': '/cfy/cloudify/plugins/diamond-plugin/plugin.yaml', 'destination_path': '/diamond-plugin/1.3rc1/plugin.yaml'}
 #  - {'source_path': '/cfy/cloudify/plugins/openstack-plugin/plugin.yaml', 'destination_path': '/openstack-plugin/1.3rc1/plugin.yaml'}
-#  - {'source_path': '/cfy/cloudify/plugins/aws-plugin/plugin.yaml', 'destination_path': '/aws-plugin/1.3rc1/plugin.yaml'}
```
Comment (ran-z) [inline openstack-manager-blueprint-inputs.yaml]: talked with Idan, he does want all the IaaS plugins to be in the inputs section (commented out), both yaml and wagon wise.. personally im on the fence so it's up to you to decide whether you want to revert this part or postpone this decision to sunday :)

File: aws-ec2-RHEL-manager-blueprint-inputs.yaml [hunk 3]
```diff
@@ -67,92 +84,150 @@ management_worker_rpm_source_url: 'https://gigaspaces-repository-eu.s3.amazonaws
...
+# By default, the Diamond, Fabric and relevant IaaS plugins are provided.
+# Note that you can upload plugins post-bootstrap via the `cfy plugins upload`
+# command.
+plugin_resources:
+ - 'https://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/rc1-RELEASE/cloudify_diamond_plugin-1.3rc1-py27-none-linux_x86_64-redhat-Maipo.tar.gz'
+ - 'https://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/rc1-RELEASE/cloudify_diamond_plugin-1.3rc1-py27-none-linux_x86_64-centos-Core.tar.gz'
+ - 'https://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/rc1-RELEASE/cloudify_diamond_plugin-1.3rc1-py26-none-linux_x86_64-centos-Final.tar.gz'
+ - 'https://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/rc1-RELEASE/cloudify_diamond_plugin-1.3rc1-py27-none-linux_x86_64-Ubuntu-precise.tar.gz'
+ - 'https://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/rc1-RELEASE/cloudify_diamond_plugin-1.3rc1-py27-none-linux_x86_64-Ubuntu-trusty.tar.gz'
+ - 'https://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/rc1-RELEASE/cloudify_fabric_plugin-1.3rc1-py27-none-linux_x86_64-redhat-Maipo.tar.gz'
```
Comment (nir0s) [inline aws-ec2-RHEL-manager-blueprint-inputs.yaml]: comment the irrelevant iaas plugin

File: vcloud-manager-blueprint.yaml [hunk 4]
```diff
@@ -449,10 +461,9 @@ inputs:
...
       - 'http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/rc1-RELEASE/cloudify_diamond_plugin-1.3rc1-py27-none-linux_x86_64-Ubuntu-precise.tar.gz'
       - 'http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/rc1-RELEASE/cloudify_diamond_plugin-1.3rc1-py27-none-linux_x86_64-Ubuntu-trusty.tar.gz'
-      - 'http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/rc1-RELEASE/cloudify_diamond_plugin-1.3rc1-py27-none-win32-none-none.tar.gz'
 
   dsl_resources:
-    description:
+    description: >
       Holds a set of dsl required resources
     default:
       - {'source_path': 'http://www.getcloudify.org/spec/tosca-vcloud-plugin/1.3rc1/plugin.yaml', 'destination_path': '/spec/tosca-vcloud-plugin/1.3rc1/plugin.yaml'}
```
Comment (nir0s) [inline vcloud-manager-blueprint.yaml]: add `broker_ip: { get_input: rabbitmq_endpoint_ip }` to agent config

**Pull request closed without merging**